functions/consumer/rabbit-consumer/src/main/java/org/springframework/cloud/fn/consumer/rabbit/RabbitConsumerProperties.java

```diff
@@ -73,6 +73,18 @@
 	 */
 	private boolean ownConnection;
 
+	/**
+	 * When mapping headers for the outbound message, determine whether the headers are
+	 * mapped before the message is converted, or afterwards. This only affects headers
+	 * that might be added by the message converter. When false, the converter's headers
+	 * win; when true, any headers added by the converter will be overridden (if the
+	 * source message has a header that maps to those headers). For example, when using a
+	 * SimpleMessageConverter with a 'byte[]' payload that contains json; if headersMappedLast
+	 * is false, the converter will set the content type to 'application/octet-stream'.
```

Comment:
This is too long for this type of javadocs.
Such a property is used as a Spring Boot metadata information and is populated for code completion assistant in IDEs (for example). So, it is really recomened to have such a description as short as possible.
More over we have respective documentation is Spring Integration on the matter: https://docs.spring.io/spring-integration/docs/current/reference/html/amqp.html#content-type-conversion-outbound. Therefore no need to be excessive in this project and its results.